libtiny_tui/src/config.rs

```diff
@@ -13,6 +13,8 @@ pub use termbox_simple::*;
 pub(crate) struct Config {
     #[serde(default)]
     pub(crate) colors: Colors,
+    #[serde(default)]
+    pub(crate) max_buffer_per_tab: Option<usize>,
```

Comment:
My renaming suggestion for this field is above.

I think we could use `usize` here and set it `usize::MAX` by default. That also simplifies the comparison below.

After renaming this let's also rename other parameter and field names so that they're all the same.